src/main/storm/mesos/MesosNimbus.java

```diff
@@ -789,15 +789,11 @@ protected void computeResourcesForSlot(final RotatingMap<OfferID, Offer> offers,
         executorInfoBuilder
             .setName(executorName)
             .setExecutorId(ExecutorID.newBuilder().setValue(details.getId()))
-            .setData(ByteString.copyFromUtf8(executorDataStr));
-        if (!subtractedExecutorResources) {
-          subtractedExecutorResources = true;
```

Comment:
Ah, thanks for pointing that out, definitely a screw up by me in removing the assignment to `subtractedExecutorResources` there and not reintroducing it in the place it used to be set to true.